examples/kiva/simple2.py

```diff
@@ -16,7 +16,8 @@ def simple():
     # directly manipulate the underlying Numeric array.
```

Comment:
^ There's still a lingering `tempfile.mktemp` call on line 14. (don't ask me why it's written that way)